src/main/java/frc/robot/Constants.java

```diff
@@ -5,13 +5,17 @@ public final class Constants {
   // Constants go here. It's standard to separate them into static inner classes. An example is created below.
 
   public static final class DriveConstants {
-
-    // Constants related to driving go here...
-
-    // This class cannot be initialized.
-    private DriveConstants() {}
+    public static final int kSpeakerShooterID1 = 19;
```

Comment:
Especially when we start to merge the swerve code, it will most likely be easier if you put these in an inner class like you did with `OIConstants`.